en/tutorials/linux_sbus.md

```diff
@@ -0,0 +1,36 @@
+# Sbus Driver for linux（drivers/linux_sbus）
+This driver allows the autopilot to fetch data from the futaba sbus receiver via serial port, up to 16 channels.  
+You can also use other receivers based on the sbus protocol, such as frsky, radioLink, and even sbus encoders.  
```

Comment:
Use capital letters along the guide please: `FrSky`, `RadioLink`, `SBUS`, `Futaba`.